### PR TITLE
chore: Replace java logging with Log4J

### LINKED ISSUE #93: Exchange java logging with Log4J
### What feature do you want to see added?

Exchange java logging with Log4J. Due Log4J being a native dependency provided by the minecraft server itself, we can utilize it with ease.

### Are there any alternatives?

No

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by NotMyFault**
## Overview

Fixes #93

## Description
This PR replaces our little bit of java logging with Log4J. The server itself provides Log4J, so we can safely use it without shading.

## Checklist

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch.
- [X] I tested my changes and approved their functionality
- [X] I ensured my changes do not break other parts of the code
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/FastAsyncVoxelSniper/blob/main/CONTRIBUTING.md)
- [X] Ensure that the pull request title represents the desired changelog entry

### CODE REVIEW

File: src/main/java/com/thevoxelbox/voxelsniper/config/VoxelSniperConfigLoader.java [hunk 1]
```diff
@@ -20,6 +22,9 @@
  */
 public class VoxelSniperConfigLoader {
 
+    protected static final String BRUSH_PROPERTIES = "brush-properties";
```
Comment (Aurelien30000) [inline src/main/java/com/thevoxelbox/voxelsniper/config/VoxelSniperConfigLoader.java:25]: Why did you move this here 🤔

**Review by Aurelien30000 — APPROVED** [review]

File: src/main/java/com/thevoxelbox/voxelsniper/config/VoxelSniperConfigLoader.java (hunk 1 above)
Comment (NotMyFault) [reply src/main/java/com/thevoxelbox/voxelsniper/config/VoxelSniperConfigLoader.java:25]: `protected` comes before `private` if you read access levels as hierarchy.

Comment (Aurelien30000) [reply src/main/java/com/thevoxelbox/voxelsniper/config/VoxelSniperConfigLoader.java:25]: Ok, makes sense!
